eggplant/profiles/templates/eggplant/profiles/profile_detail.html

```diff
@@ -1,62 +1,53 @@
 {% extends "eggplant/core/base.html" %}
 {% load staticfiles %}
+{% load i18n %}
 {% load bootstrap3 %}
 
 {% block app_css %}
-	<link href="{% static 'css/bootstrap-datepicker3.min.css' %}"
-		rel="stylesheet" />
+<link href="{% static 'css/bootstrap-datepicker3.min.css' %}" rel="stylesheet" />
 {% endblock%}
+
 {% block app_js %}
-	<script src="{% static 'js/bootstrap-datepicker.min.js' %}"></script>
-	<script src="{% static 'js/locales/bootstrap-datepicker.da.min.js' %}"></script>
-	<script src="{% static 'js/locales/bootstrap-datepicker.en-GB.min.js' %}"></script>
-	<script type=text/javascript>
+<script src="{% static 'js/bootstrap-datepicker.min.js' %}"></script>
+<script src="{% static 'js/locales/bootstrap-datepicker.da.min.js' %}"></script>
+<script src="{% static 'js/locales/bootstrap-datepicker.en-GB.min.js' %}"></script>
+<script type=text/javascript>
 var $jq = jQuery.noConflict();
 $jq(function() {
     {% if form.instance %}
     $jq('#dob_datepicker')
-    	.val('{{form.instance.end_date|date:"Y-m-d"}}')
-    	.datepicker({format:'yyyy-mm-dd'});
+      .val('{{form.instance.end_date|date:"Y-m-d"}}')
+      .datepicker({format:'yyyy-mm-dd'});
     {% else %}
     $jq('#dob_datepicker').datepicker({format:'yyyy-mm-dd'});
     {% endif %}
 });
-	</script>
+</script>
 {% endblock%}
 
 {% block content %}
-
-
 <div class="container">
-<div class="row">
-<div class="col-md-5 col-md-offset-3">
-
-{% if user.profile.is_complete %}                
-	<h3>Your details</h3>
-{% else %}
-	<h3>Please update Your details</h3>
-{% endif %}
-
-<form enctype="multipart/form-data" action="{% url 'eggplant:profiles:profile' %}" method="post">
-    {% csrf_token %}
-	{% bootstrap_form form %}
-	{% buttons %}
-	  <button type="submit" class="btn btn-primary">
-	    {% bootstrap_icon "star" %} Submit
-	  </button>
-	{% endbuttons %}
-</form>
-
-<hr/>
-
-</div>
-</div>
-{% if user.profile.is_complete %}
-<div class="row">
-<div class="col-md-5 col-md-offset-3">
-	<a class="btn btn-primary" href="{% url 'account_change_password' %}">Change Password</a>
-</div>
-</div>
-{% endif %}
+    <div class="row">
+        <div class="col-md-6 col-md-offset-3">
+            {% if user.profile.is_complete %}
+              <h3>{% trans 'Your details' %}</h3>
```

Comment:
Thanks for adding trans blocks, there really has to be a 100% coverage unless we want the Danish end-result to look... weird...